releases/SF30_Pro-SN30_Pro/1.23/sf30sn30pro.metainfo.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 Richard Hughes <richard@hughsie.com> -->
+<!-- Copyright 2018 Mario Limonciello <mario.limonciello@dell.com> -->
+<component type="firmware">
+  <id>com.8bitdo.sf30sn30pro.firmware</id>
+  <name>SF30/SN30 Pro</name>
+  <summary>Firmware for the 8Bitdo SF30/SN30 Pro Game Controller</summary>
+  <description>
+    <p>
+      Updating the firmware on your SF30/SN30 Pro device improves performance and adds
+      new features.
+    </p>
+  </description>
+  <provides>
+    <!-- USB\VID_057E&PID_2009 -->
+    <firmware type="flashed">b3850dd5-a010-5d5c-8834-2ff809e95869</firmware>
+    <!-- USB\VID_2DC8&PID_6000 -->
+    <firmware type="flashed">269b3121-097b-50d8-b9ba-d1f64f9cd241</firmware>
+    <!-- USB\VID_2DC8&PID_6001 -->
+    <firmware type="flashed">81becceb-f87b-5c66-b30a-b0958043d65d</firmware>
+    <!-- USB\VID_045E&PID_028E -->
+    <firmware type="flashed">e5f6e0cf-c156-58e6-94e5-10ef2dff807c</firmware>
+    <!-- USB\VID_054C&PID_05C4 -->
+    <firmware type="flashed">50afc861-c345-56d2-b4be-8a54e6a1ed12</firmware>
+  </provides>
+  <url type="homepage">http://www.8bitdo.com/sn30pro-sf30pro/</url>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Proprietary</project_license>
+  <developer_name>8Bitdo</developer_name>
+  <releases>
+    <release urgency="medium" version="1.23" date="2017-12-28">
+      <checksum filename="Firmware_SF30&amp;SN30_Pro_V1.23.dat" target="content"/>
+      <description>
+        <p>
+          This stable release fixes the following issues:
+        </p>
+        <ul>
+          <li>1.Enhanced vibration optimization on Xinput mode.</li>
```

Comment:
No need for `1.` (it's an unordered list..) and no need for the trailing dot.

Reply:
OK